src/general/block-endpoint-edit-role/src.rego

```diff
@@ -0,0 +1,32 @@
+package blockendpointeditrole
+
+violation[{"msg": msg}] {
+    input.review.object.metadata.name == "system:aggregate-to-edit"
+    endpointRule(input.review.object.rules[_])
+    msg := "ClusterRole system:aggregate-to-edit should not allowed endpoint permissions"
+}
+
+violation[{"msg": msg}] {
+    input.review.object.metadata.name == "system:aggregate-to-edit"
+    not disableAutoupdate(input.review.object.metadata.annotations)
```

Comment:
Might be easier/faster to avoid the function definition and just write:

`annotations["rbac.authorization.kubernetes.io/autoupdate"] == "false"`

directly here.

Reply:
removed annotation validation, and only leave advise in violation msg